Cargo.toml

```diff
@@ -24,7 +26,7 @@ include = [
 path = "bindings/rust/lib.rs"
 
 [dependencies]
-tree-sitter = "0.21.0"
+tree-sitter = "0.22.5"
```

Comment:
@ObserverOfTime, this update seems to be what fixed the `Cursor` bug https://github.com/tree-sitter/tree-sitter/issues/2767

Do you happen to know which tree-sitter PR it was that fixed it? Just curious!

Reply:
No idea.